src/pages/nodes/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-title: Validate
-description: Become one of the important pieces of ZetaChain operation by validating transactions for the chain. It’s easy to get setup and started on our testnet and mainnet.
-heroImgUrl: /img/pages/validate.svg
+title: Operate
+description: Become one of the important pieces of ZetaChain operation by running your own nodes. It’s easy to get setup and started on our Testnet and Mainnet.
```

Comment:
"Become one of the important pieces of ZetaChain operation by validating transactions for the chain." - > "Become an important part of the ZetaChain network by operating an node. It's easy to get started on Testnet or Mainnet."